web_export_view/static/src/js/web_export_view.js

```diff
@@ -16,8 +16,12 @@ odoo.define('web_export_view', function (require) {
             var self = this;
             this._super.apply(this, arguments);
             if (self.getParent().renderer.viewType == 'list') {
-                self.$el.find('.o_dropdown').last().append(QWeb.render('WebExportTreeViewXls', {widget: self}));
-                self.$el.find('.export_treeview_xls').on('click', self.on_sidebar_export_treeview_xls);
+                session.user_has_group('web_export_view.group_disallow_export_view_data_excel').then(function (has_group) {
```

Comment:
Why calling again `has_group` if you have already called `user_has_group`?

Reply:
No, `user_has_group` returns a deferred object that we need to resolve in the proper method. `has_group` is the boolean value we're evaluating after all.

Reply:
Sorry for the noise. My lack of JS knowledge...